conf/application.conf

```diff
@@ -142,7 +142,7 @@ microservice {
       port = 8500
     }
     email {
-      host = localhost
```

Comment:
this should remain as localhost within the src code and be overridden to email.public.mdtp in app-config-base

Reply:
Comment addressed, please check